ecosystem/sep-0024.md

```diff
@@ -214,12 +288,12 @@ banking rails so counterparties can run through the flow without a need to colla
 - For each anchor, use information from its [`stellar.toml`](sep-0001.md) file and its [`/info`](#info) endpoint to
   display useful information to the user about the asset they've picked.
 - Provide a UI that allows users to pick an asset, anchor, and amount to use for deposit or withdraw. The UI should
-  display the asset's fee structure (if possible) as well as information such as the address of the anchor and
-  description of the asset from the `stellar.toml` file.
-- **Use the `/info` endpoint**
-  - Fetch the asset's deposit & withdawal fee structure: if `fee_fixed`, `fee_percent`, or `fee_minimum` are provided,
-    show this to the user early in the process so they're fully informed.
-  - If the `/fee` endpoint is enabled, use it for computing fees when you need to show them to the user.
+  display the asset's quote (if possible) as well as information such as the address of the anchor and description of
+  the asset from the `stellar.toml` file.
+- **If [SEP-38] quote API is supported by the anchor**
+  - Fetch the asset's exchange rate at the market price using [`SEP-38 GET /price`](sep-0038.md#get-price) API.
+  - For the firm quote, use [`SEP-38 POST /quote`](sep-0038.md#post-quote) API and show the quote and expiration time to
```

Comment:
I am not sure of handling the firm quote before the UI flow starts. I agree getting the indicative quote may be helpful for the wallet. However if some of the SEP-24 anchors are handling the exchange in the UI flow, wouldn't this cause divergence in how the exchanges are handled?

Reply:
Anchors internally have a firm quote. The idea is moving firm quote from the interactive UI into the wallet UI.